sdks/python/apache_beam/internal/apiclient.py

```diff
@@ -117,7 +117,7 @@ def __init__(self, packages, options, environment_version):
     self.proto.userAgent = dataflow.Environment.UserAgentValue()
     self.local = 'localhost' in self.google_cloud_options.dataflow_endpoint
 
-    sdk_name, version_string = get_sdk_name_and_version()
+    version_string = version.__version__
```

Comment:
This reversion looks like a bad merge.

`get_sdk_name_and_version()` would return `version.__version__` and it will also use it to set the sdk name correctly.